### PR TITLE
fix(UI-1074): truncate and display sessions table columns with ellipsis

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Truncate all columns:

image

## Linear Ticket
https://linear.app/autokitteh/issue/UI-1074/truncate-and-display-sessions-table-columns-with-ellipsis

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [x] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/atoms/table/tD.tsx
Comment (J1za) [inline src/components/atoms/table/tD.tsx:1]: image

Comment (RonenMars) [reply src/components/atoms/table/tD.tsx:1]: fixed

**Review by J1za — APPROVED** [review]

**Comment by RonenMars** [comment]
@J1za Please review the last commit

File: src/components/organisms/deployments/sessions/tableRow.tsx
Comment (J1za) [inline src/components/organisms/deployments/sessions/tableRow.tsx:1]: image
align

Comment (RonenMars) [reply src/components/organisms/deployments/sessions/tableRow.tsx:1]: fixed here: https://github.com/autokitteh/web-platform/pull/761/commits/80f0d473d86e833aac12630bc4c0119f25b39263

**Review by J1za — APPROVED** [review]

**Pull request merged**